Sources/Core/Extensions/Other/SKStoreReviewController+PovioKit.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SKStoreReviewController+PovioKit.swift
```

Comment:
Based on the previous files this should be `SKStoreReviewController+Povio.swift` , or do we stick to this new style?

Reply:
I see there is mixed naming convention taking place already. I'll update in a separate PR.